app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PostsController < ApplicationController
+  def index
+    # @users = User.all
+    # @user = @users.find(params[:user_id])
+    # @posts = @user.posts
+  end
+
+  def show
+    # @users = User.all
+    # @user = @users.find(params[:user_id])
+    # @post = @user.posts.find(params[:id])
```

Comment:
- [ ] Please remove any commented code from your codebase because it's bad coding practice to leave commented(unused) code.